package.json

```
@@ -38,22 +38,23 @@
     "add": "^2.0.6",
     "axios": "^0.21.0",
     "axios-mock-adapter": "^1.19.0",
-    "babel-plugin-module-resolver": "^4.0.0",
     "json-stable-stringify": "^1.0.1",
+    "babel-plugin-module-resolver": "^4.1.0",
+    "jest-styled-components": "^7.0.3",
     "jwt-decode": "^3.1.2",
     "metro-react-native-babel-preset": "^0.66.2",
     "moment": "^2.29.1",
     "prop-types": "^15.7.2",
     "punycode": "^2.1.1",
     "react": "16.13.1",
```

[RNUH](https://react-native-community.github.io/upgrade-helper/?from=0.64.1&to=0.66.2) recommends bumping react to 17.0.2.
```suggestion
    "react": "17.0.2",
```

---

Done